sharepointembedded-chatembedded-react/src/ChatEmbeddedAPI.ts

```diff
@@ -44,6 +44,12 @@ export interface ChatLaunchConfig {
     zeroQueryPrompts?: IZeroQueryPrompts;
     suggestedPrompts?: string[];
     instruction?: string;
+    locale?: string;
+    /**
+   * String to be used as placeholder text for chat input
+   * @defaultValue 'Ask questions or type / to add files or people.'
+   */
+    chatInputPlaceholder?: string;
```

Comment:
Given that we're disabling the Files panel, should we change the default placeholder if the SPE Dev hasn't set it themselves? To something like `Ask questions or type / to add people.`